package-lock.json

```diff
@@ -5523,9 +5592,6 @@
       "integrity": "sha1-bUUk6LlV+V1PW1iFHOId1y+06VI=",
       "dev": true
     },
-    "magnific-popup": {
```

Comment:
Not sure how this one got here in the first place, but this would have been why the build was failing before without changing the Sass import paths